graphql/util.go

```diff
@@ -1307,3 +1307,26 @@ func applyVolumeOptions(ctx context.Context, volume host.Volume, volumeOptions r
 	}
 	return nil
 }
+
+func setVersionActivationStatus(sc data.Connector, version *model.Version) error {
+	defaultSort := []task.TasksSortOrder{
+		{Key: task.DisplayNameKey, Order: 1},
+	}
+	opts := data.TaskFilterOptions{
+		Sorts: defaultSort,
+	}
+	tasks, _, err := sc.FindTasksByVersion(version.Id, opts)
+	if err != nil {
+		return errors.Wrapf(err, "error getting tasks for version %s", version.Id)
+	}
+	if !task.AnyActiveTasks(tasks) {
+		if err = version.SetNotActivated(); err != nil {
```

Comment:
simplification suggestion: can just return `errors.Wrapf(version.SetNotActivated, "Error updating version activated status for '%s'", version.Id`, i.e. can remove the second Sprintf (handled by Wrapf) and just return regardless of error, since Wrapf will return nil if err is nil. 

Then, we can skip the else if entirely (since everything else falls into this case) and just `return errors.Wrapf(version.SetActivated, ....)`